src/main/resources/lang/default.lang

```diff
@@ -1613,6 +1667,23 @@ visual effects:
 			name: scrape @-
 			pattern: scrape
 
+		# 1.19 particles
+		sonic_boom:
+			name: sonic boom @-
+			pattern: sonic boom
+		sculk_soul:
+			name: sculk soul @-
+			pattern: sculk soul
+		sculk_charge:
+			name: sculk charge @-
+			pattern: sculk charge
+		sculk_charge_pop:
+			name: sculk charge pop @-
+			pattern: sculk charge pop
+		shriek:
+			name: skriek @-
```

Comment:
```suggestion
			name: shriek @-
```

Reply:
I stared at this for a solid 2 minutes before realizing what you changed.  Haha.